src/mpi/romio/adio/ad_gpfs/ad_gpfs_aggrs.c

```diff
@@ -677,12 +677,14 @@ void ADIOI_GPFS_Calc_others_req(ADIO_File fd, MPI_Count count_my_req_procs,
 #ifdef AGGREGATION_PROFILE
     MPE_Log_event(5026, 0, NULL);
 #endif
-    /* Send 1 int to each process.  count_my_req_per_proc[i] is the number of
+    /* Preliminary "how much work will we do?" data exchange:
+     * count_my_req_per_proc[i] is the number of
      * requests that my process will do to the file domain owned by process[i].
-     * Receive 1 int from each process.  count_others_req_per_proc[i] is the number of
+     * count_others_req_per_proc[i] is the number of
      * requests that process[i] will do to the file domain owned by my process.
-     */
-    count_others_req_per_proc = ADIOI_Malloc(nprocs * sizeof(int));
+     * Just sending/recieving one value here: a few lines later we'll set up
```

Comment:
```suggestion
     * Just sending/receiving one value here: a few lines later we'll set up
```

Reply:
i never get that right... glad the spell-checker is working